modules/error_log/utils.py

```diff
@@ -0,0 +1,28 @@
+import discord
+import discord.utils
+import os
+import csv
+from datetime import datetime
+
+def get_discord_obj(iterable, label: str):
```

Comment:
Instead of copy pasting this function from another module, consider making a folder `/utils` or `/lib` where you can put this function and import it from within any module